### ISSUE TITLE
depth 解码器是否有问题

### ISSUE THREAD

**Issue by ShifaTang** [VDT-2048/SAINet#3]
img_0, img_1, img_2, img_3, img_4 = self.layer_rgb(imgs)
        dep_0, dep_1, dep_2, dep_3, dep_4 = self.layer_dep(depths)

        # RGB/D Decoder #
        ###############################################
        x_rgb_42 = self.rgb_gcm_4(img_4)
        x_rgb_32 = self.rgb_gcm_3(torch.cat((img_3, self.upsample_2(x_rgb_42)), dim=1))
        x_rgb_22 = self.rgb_gcm_2(torch.cat((img_2, self.upsample_2(x_rgb_32)), dim=1))
        x_rgb_12 = self.rgb_gcm_1(torch.cat((img_1, self.upsample_2(x_rgb_22)), dim=1))
        x_rgb_02 = self.rgb_gcm_0(torch.cat((img_0, self.upsample_2(x_rgb_12)), dim=1))

        x_dep_42 = self.dep_gcm_4(img_4)
        x_dep_32 = self.dep_gcm_3(torch.cat((img_3, self.upsample_2(x_dep_42)), dim=1))
        x_dep_22 = self.dep_gcm_2(torch.cat((img_2, self.upsample_2(x_dep_32)), dim=1))
        x_dep_12 = self.dep_gcm_1(torch.cat((img_1, self.upsample_2(x_dep_22)), dim=1))
        x_dep_02 = self.dep_gcm_0(torch.cat((img_0, self.upsample_2(x_dep_12)), dim=1))
        
  这里的self.dep_gcm_4(img_4) 是否应该换成self.dep_gcm_4(dep_4)，否则与论文中的结构图有差异
